core/src/test/java/nl/nn/adapterframework/pipes/Base64PipeTest.java

```diff
@@ -156,7 +157,7 @@ public void inputStringOutputStream() throws ConfigurationException, PipeStartEx
 		pipe.start();
 
 		PipeRunResult prr = doPipe(pipe,input, session);
-		InputStream result = (InputStream) prr.getResult().asObject();
```

Comment:
Het idee achter asObject was om te controlleren of er daadwerkelijk een stream uitkomt ipv dat `Message` het converteerd naar een stream